bigframes/session.py

```diff
@@ -97,6 +97,8 @@
 _BIGQUERYCONNECTION_REGIONAL_ENDPOINT = "{location}-bigqueryconnection.googleapis.com"
 _BIGQUERYSTORAGE_REGIONAL_ENDPOINT = "{location}-bigquerystorage.googleapis.com"
 
+_BIGFRAMES_DEFAULT_CONNECTION_ID = "bigframes-default-connection"
```

Comment:
Should we add a test that captures the use of this? Will have to create the connection and  configure IAM on its  SA as a prerequisite.

Reply:
I can add one after http://cl/571816142.

Reply:
Added